app/src/main/java/com/github/epfl/meili/storage/FirebaseStorageService.kt

```diff
@@ -0,0 +1,21 @@
+package com.github.epfl.meili.storage
+
+import android.net.Uri
+import com.google.firebase.storage.FirebaseStorage
+
+object FirebaseStorageService: StorageService {
+
+    private val firebaseStorage = FirebaseStorage.getInstance()
+
+    override fun uploadFile(remotePath: String, filePath: Uri, onSuccessCallback: () -> Unit, onFailureCallback: () -> Unit) {
```

Comment:
Shouldn't we compress the image before storing it in firebase?

Reply:
Good idea, done!